x-pack/plugins/synthetics/common/rules/synthetics/translations.ts

```diff
@@ -33,6 +33,16 @@ export const SyntheticsMonitorStatusTranslations = {
       },
     }
   ),
+  defaultRecoverySubjectMessage: i18n.translate(
```

Comment:
@florent-leborgne 

The translations you reviewed are all part of the legacy Uptime app. It would be helpful if you could review the translations in this file, which contain our new translations. I would honestly be extremely grateful if you reviewed the entire file, as we've discussed internally that the content needs work https://github.com/elastic/kibana/blob/56f637b82edf81b7ab6dafd54c197f1b945c487b/x-pack/plugins/synthetics/common/rules/synthetics/translations.ts

Reply:
I'd happily do that. What do you think should be improved in this file? I have a few ideas to simplify but I'd also like to know what you've discussed internally :)

Reply:
I think one of the things that came up internally was that the subject line for email connectors read weird. https://github.com/elastic/kibana/blob/56f637b82edf81b7ab6dafd54c197f1b945c487b/x-pack/plugins/synthetics/common/rules/synthetics/translations.ts#L29. It's different than the legacy Uptime one. Perhaps I should take a look at your feedback for the legacy Uptime subject line. Personally `The monitor` vs `Your monitor` sounds strange to me.

Ultimately though, these translations were merged in without any content review so we're kinda behind and want to make sure it gets reviewed before GA.

Reply:
I can try to set up some time to compare them side by side and make them more consistent with each other? I'd just need to know the precise files to compare and tweak to submit a PR, if you can help me with that. I'd hopefully be able to do that by the end of this week or early next week.

Reply:
@florent-leborgne Sure thing. A few things to clarify though:

1. I don't think we're concerned with them being more consistent. Uptime is going away and if there are better translations for Synthetics, we're happy to go with that without updating Uptime. 
2. I see when you reviewed Uptime, you did use `The monitor` in the subject line, which is what Synthetics has currently. I'm happy to go with this as I was likely the only one who thought it sounded weird and it's clear to me now that the phrase is approved by you.
3. Given that we don't need consistency between Uptime and Synthetics, you don't need to put up a PR to make the two consistent if you don't feel it's necessary, so long as the `synthetics/translations.ts` file meets your standards we're happy.
4. If you do still want to compare and put up a PR, the two files are https://github.com/elastic/kibana/blob/56f637b82edf81b7ab6dafd54c197f1b945c487b/x-pack/plugins/synthetics/common/rules/synthetics/translations.ts#L29 and https://github.com/elastic/kibana/blob/56f637b82edf81b7ab6dafd54c197f1b945c487b/x-pack/plugins/synthetics/common/rules/legacy_uptime/translations.ts (Please note, there are more translations in legacy_uptime/translations because there are more rule types in Uptime)

As far as what I'll do with this PR moving forward:

1. The translations you originally reviewed are actually from legacy Uptime and have been established for a while now. I'm not 100% sure if we want to change them. I'll go ahead and check with our team lead and if he approves I will apply those changes to legacy Uptime, otherwise I will not apply the changes.
2. Since you are volunteering to put up a PR and don't have any pressing suggestions for the `synthetics/translations.ts` file, I will go ahead and merge this within the next few hours (need to do some more testing myself) and allow you to put up a secondary PR (if you desire, I mentioned above that may not be necessary).

Reply:
Ok, I reviewed all content I could find in the PR. Feel free to not apply my suggestions if those are for the part of the feature that is going away soon :)

I'm pretty OK with what I saw in the translations file, so we could maybe keep it this way for now, until we get more data or more time to maybe try to simplify it? I'm totally ok to replace "The monitor" with "Your monitor". It also sounds good to me. 

Let me know what you think.